atoma-state/src/handlers.rs

```diff
@@ -671,6 +671,49 @@ pub(crate) async fn handle_state_manager_event(
                 .update_stack_total_hash(stack_small_id, total_hash)
                 .await?;
         }
+        AtomaAtomaStateManagerEvent::GetStacksForModel {
```

Comment:
All looks good, I suggest adding tracing messages, especially on errors. I should do the same for the atoma node logic.